scripts/cleanup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash 
+
+namespaces=(default non-default)
+
+for n in ${namespaces[@]}; do
+   echo "********** Namesapce:" $n "******************"
+   kubectl get all -n $n -l 'testing.kubernetes.io in (true)' | grep -v NAME | awk '{print $1}' | xargs kubectl delete -n $n
+   kubectl get serviceexports -n $n -l 'testing.kubernetes.io in (true)' | grep -v NAME | awk '{print $1}' | xargs kubectl delete serviceexport -n $n
+   kubectl get serviceimports -n $n -l 'testing.kubernetes.io in (true)' | grep -v NAME | awk '{print $1}' | xargs kubectl delete serviceimport -n $n
+   kubectl get httproutes -n $n -l 'testing.kubernetes.io in (true)' | grep -v NAME | awk '{print $1}' | xargs kubectl delete httproute -n $n
+   kubectl get gateway -n $n -l 'testing.kubernetes.io in (true)' | grep -v NAME | awk '{print $1}' | xargs kubectl delete gateway -n $n
+done
```

Comment:
Do you need to clean service, deployment and grpcroutes?

Reply:
+1